menu.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+  <title>Tsige's Personal Portifolio</title>
+</head>
+
+<body>
+  <section class="headline">
+   <div class="works">
+      <h2>My Recent Works</h2>
+      <img src="images/icons/Indicator.svg" alt="Indicator"><br>
+    <div class="work-list">
+      <div class="grid-container1">
+        <div class="grid-item1"></div>
+        <div class="grid-item2">
+          <h4>Multi-Post Stories Gain+Glory</h4>
+          <div id="2">
+            <ul>
+              <li>Ruby On Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+          </div> 
+          <div class="button">See project</div>
+      </div> 
+        </div>
+      <div class="grid-container1">
+        <div class="grid-item1"></div>
+        <div class="grid-item2">
+          <h4>Multi-Post Stories Gain+Glory</h4>
+          <div id="2">
+            <ul>
+              <li>RubyOnRails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+          </div> 
+          <div class="button">See project</div>
```

Comment:
- [ ] As I suggested in the previous review according to the design, This should be a link `a` or `button` tag, and it should follow the button styling: `Active, Hover and Pressed`  I'll suggest you update this part of your code according. :+1: :smile: 
- [ ] I can see you have added the CSS for the hover state. :+1: But I think the color of your hover state does not match the given design. Please go to the Figma design and get the correct colors. :+1: :smile: 

- FIGMA DESIGN BUTTON STATES:
![Screenshot from 2023-05-06 18-19-47](https://user-images.githubusercontent.com/107632780/236638398-0f809540-85e4-41af-85a4-295955aad693.png)